### PR TITLE
next meeting + poll proposal

### PR DESCRIPTION
**Pull request by conrad-watt**
Realised this hadn't been created yet. Happy to push to next meeting if this is too near to propose a poll.

### CODE REVIEW

**Comment by binji** [comment]
Ah, sorry about this. Yeah, I'd say this is too soon for a poll (I believe last time we discussed this we required a few days at least). Let's push to the November 10th meeting and cancel tomorrow's meeting.

**Comment by conrad-watt** [comment]
I can edit this PR to point the meeting at Nov 10 if that's easiest.

**Comment by binji** [comment]
Sounds good. Thanks, Conrad!

**Comment by conrad-watt** [comment]
force-pushed

File: main/2020/CG-11-10.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the November 10th video call of WebAssembly's Community Group
+
+- **Where**: zoom.us
+- **When**: November 10th, 4pm-5pm UTC (November 10th, 9am-10am Pacific Daylight Time)
```
Comment (binji) [inline main/2020/CG-11-10.md:6]: I think this will be 5pm UTC, 9am PST (DST ends on Nov 1st)

Comment (conrad-watt) [reply main/2020/CG-11-10.md]: ah, fixed (on the assumption that the PT time is fixed at 9am)

**Review by binji — APPROVED** [review]

**Pull request merged**